lib/pharos/ssh/remote_command.rb

```diff
@@ -115,7 +115,7 @@ def run
       def initialize_debug
         if self.class.debug?
           @debug = true
-          @pastel = Pastel.new
+          @pastel = Pastel.new($stdout.tty?)
```

Comment:
Why this fix is directly with `$stdout`?

Reply:
Because it's not aware of the command.

More reasons to have a global `configure`able output helper thing.